core/src/main/kotlin/com/malinskiy/marathon/analytics/internal/pub/Track.kt

```diff
@@ -0,0 +1,52 @@
+package com.malinskiy.marathon.analytics.internal.pub
+
+import com.malinskiy.marathon.device.Device
+import com.malinskiy.marathon.device.DeviceInfo
+import com.malinskiy.marathon.device.DevicePoolId
+import com.malinskiy.marathon.execution.TestResult
+import java.time.Instant
+import java.util.concurrent.atomic.AtomicReference
+
+class Track : Tracker {
+    private val delegates: AtomicReference<MutableList<Tracker>> = AtomicReference(mutableListOf())
+
+    operator fun plus(tracker: Tracker): Track {
+        delegates.getAndUpdate { list ->
```

Comment:
I'm not sure about thread safety here

Reply:
the population happens in a single thread anyways, but even if not - what's the problem?